qa/workunits/rbd/test_librbd.sh

```diff
@@ -2,7 +2,7 @@
 
 if [ -n "${VALGRIND}" ]; then
   valgrind --tool=${VALGRIND} --suppressions=${TESTDIR}/valgrind.supp \
-    ceph_test_librbd
+    --error-limit=no --error-exitcode=1 ${VALGRIND_ARGUMENTS} ceph_test_librbd
```

Comment:
maybe make it a large exit code to distinguish it from non-valgrind failures easily in job summaries